addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -137,7 +137,9 @@ var SnippetEditor = Widget.extend({
      * @override
      */
     destroy: function () {
+        const $optionsSection = this.$el.data('$optionsSection');
         this._super(...arguments);
+        $optionsSection.remove();
```

Comment:
Probably the same bug in 13.0, so you can make a dedicated commit in another PR. Also instead of using `.data('uglyStuff')`, you can remove the last element in `this._customize$Elements`.

Reply:
Doesn't `_customize$Elements` also contain the parent's `$optionsSection`?

Reply:
`_customize$Elements` is a list containing the parent ones and the current one. The last one is the current one.

Reply:
Makes sense, will make a PR for that in 13.0 and link it here.